db/migrate/20150430132128_update_projects_defaults_and_indexes.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateProjectsDefaultsAndIndexes < ActiveRecord::Migration
+  def up
+    change_column :projects, :approved, :boolean, default: false
+    add_index :projects, :approved
+    change_column :projects, :beta, :boolean, default: false
+    add_index :projects, :beta
+  end
+
+  def down
+    change_column :projects, :approved, :boolean, default: nil
+    change_column :projects, :beta, :boolean, default: nil
```

Comment:
This is fine, but in the future you can have it all under a def change method using `change_column_default` like `change_column_default(:projects, :approved, false)`